src/components/Card/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import {activeOpacity, metrics} from '../../helpers/metrics'
+import type {ITheme} from '../../theme'
+import type {ViewProps} from 'react-native'
+
+export type CardProps = {
+  onPress?: () => void
+  children?: React.ReactNode
+  bgColor?: string
+  style?: ViewProps
+  testId?: string
```

Comment:
do we need to pass testId props ?

Reply:
i think we need pass testId  props because maybe we have to use many Card in one Screen